core/src/avm2/globals/flash/text/engine/ElementFormat.as

```diff
@@ -199,5 +203,13 @@ package flash.text.engine {
         public function set typographicCase(value:String):void {
             this._typographicCase = value;
         }
+
+        public function getFontMetrics():FontMetrics {
+            stub_method("flash.text.engine.ElementFormat", "getFontMetrics");
+            var emBox:Rectangle = new Rectangle(0, _fontSize * -0.8, _fontSize, _fontSize);
+            return new FontMetrics(
+                emBox, -5, 1.2, 1.8, 1.2, 0.075, 0.6, -0.35, 0.6, 0.0
```

Comment:
are these completely random numbers, or taken from some existing font? :D

Reply:
Those are numbers taken from an existing font (roughly). I don't think their values matter, but their signs and orders of magnitude might.